libnetwork/network.go

```diff
@@ -175,7 +175,8 @@ func (i *IpamInfo) UnmarshalJSON(data []byte) error {
 type Network struct {
 	ctrlr            *Controller
 	name             string
-	networkType      string // networkType is the name of the netdriver used by this network
+	networkType      string               // networkType is the name of the netdriver used by this network
+	driverCaps       driverapi.Capability // driverCaps is a list of capabilities the associated networkType has.
```

Comment:
I think `n.driverCaps` needs to be included in the `MarshalJSON`/`UnmarshalJSON`/`CopyTo`?